spirv_glsl.cpp

```diff
@@ -366,6 +366,14 @@ void CompilerGLSL::find_static_extensions()
 
 			if (type.basetype == SPIRType::Half)
 				require_extension_internal("GL_AMD_gpu_shader_half_float");
+
+			if (type.basetype == SPIRType::Int || type.basetype == SPIRType::UInt)
```

Comment:
Hm, this is inconsistent with how FP16 was implemented, using Half, and 64-bit types.
Either we should rename Half to Float and use width, or add SByte/SByte/Short/UShort as new types.

I think I prefer the latter, because it avoids problems where we see Int/UInt but forget to check for width explicitly and it makes failures to take width into account more obvious because we end up hitting unknown types.